tcn_hpl/data/utils/ptg_datagenerator.py

```diff
@@ -24,6 +24,23 @@
 from angel_system.data.medical.data_paths import LAB_TASK_TO_NAME
 
 
+TASK_TO_NAME = {
+    "m1": "M1_Trauma_Assessment",
+    "m2": "M2_Tourniquet",
+    "m3": "M3_Pressure_Dressing",
+    "m4": "M4_Wound_Packing",
+    "m5": "M5_X-Stat",
+    "r18": "R18_Chest_Seal",
+}
+
+
+def bbn_to_dive(raw_data_root, dive_output_dir, task, label_mapping, label_version=1):
+    from angel_system.data.medical.load_bbn_data import bbn_activity_txt_to_csv
```

Comment:
Is this an "optional" import? Either way, I would suggest that imports be at the top of the file, wrapped in a try-catch if they are conditional imports with appropriate handling.

Reply:
it is conditional to whether activity_gt exists in the dataset. we can only check if that exists after the dataset is defined with the kwcoco file.

Reply:
I'm specifically focused on the `import` in this function. I'm trying to understand if there is a reason for the `from ... import ...` can not be up with the other imports in the file.

Something I just noticed is that this is possibly a circular import w.r.t. the `angel_system` package (which depends on this `tcn_hpl). Could you confirm if this is an issue? If this is why we are putting this import inside this function, please documenting it as much with a comment.

Reply:
that particular function does not depend on tcn_hpl. I have no objections to calling it at the top.